source/plugins/go/input/lib/kubernetescontainerinventory.go

```diff
@@ -182,6 +159,58 @@ func GetContainerInventoryRecords(podItem map[string]interface{}, batchTime stri
 	return containerInventoryRecords
 }
 
+func addImageInfoToContainerInventoryRecord(containerInventoryRecord map[string]interface{}, imageValue string) {
+	// image can be in any one of below formats:
+	// repository/image[:imagetag | @digest], repository/image:imagetag@digest, repo/image, image:imagetag, image@digest, image
+	if imageValue == "" {
+		return
```

Comment:
code looks good to me. But, lets test this with all possible combinations to ensure no regressions and also we dont have any bugs. This would be good opportunity for you to learn as well.